### PR TITLE
docs: Update README.md with Angular MDC framework

### PR DESCRIPTION
**Pull request by trimox**
Add [Angular MDC](https://github.com/trimox/angular-mdc-web) to recommended frameworks.

### CODE REVIEW

**Comment by touficbatache** [comment]
Should we also add [MDC-WP](https://github.com/touficbatache/material-components-web-wordpress)? I'm still working on it :smiley:

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Material Components for the web is the successor to [Material Design Lite](https
   - [Preact Material Components](https://github.com/prateekbh/preact-material-components)
   - [RMWC: React Material Web Components](https://github.com/jamesmfriedman/rmwc)
   - [Blox Material](https://blox.src.zone/material): Angular Integration Library.
+  - [Angular MDC](https://github.com/trimox/angular-mdc-web)
```
Comment (amrtgaber) [inline README.md]: Looks great! Can you insert this one line above, between lines 16 and 17 instead of lines 17 and 18?

**Comment by amrtgaber** [comment]
@touficbatache That is an interesting project, however, this pull request is to review angular-mdc, if you'd like for us to take a look at your wordpress integration please create a separate pull request.

File: README.md (hunk 1 above)
Comment (trimox) [reply README.md]: Appreciate the review @amrtgaber! All set.

**Pull request merged**